pub-critic/src/app/api/GamesShared.ts

```diff
@@ -0,0 +1,54 @@
+export const gamesApi = "https://api.rawg.io/api";
+export const apiKey = "464bc085dbbf4f33bcb2ccb39d36a6ec";
+
+export interface Screenshot {
+  id: number;
+  image: string;
+}
+
+export interface Genre {
+  id: number;
+  name: string;
+  slug: string;
+  games_count: number;
+  image_background: string;
+}
+
+export interface PlatformFull {
+  platform: Platform;
+  released_at: string;
+}
+
+export interface Platform {
+  id: number;
+  name: string;
+  slug: string;
+  image: string;
+  year_end: number;
+  year_start: number;
+  games_count: number;
+  image_background: string;
+}
+
+export interface Tag {
+  id: number;
+  name: string;
+  slug: string;
+  language: string;
+  games_count: number;
+  image_background: string;
+}
+
+export interface Game {
+  id: number;
+  slug: string;
+  name: string;
+  released: string;
+  background_image: string;
+  metacritic: number;
+  rating: number;
+  short_screenshots: Screenshot[];
+  tags: Tag[];
+  genres: Genre[];
+  platforms: PlatformFull[];
+}
```

Comment:
create `app/interfaces` folder and move all common interfaces there

Reply:
:white_check_mark: